coding-guidelines/python.md

```diff
@@ -0,0 +1,159 @@
+Coding Style and Guidelines
+===========================
```

Comment:
Could you please use the '#' for headers? That would keep it consistent with other club Markdown files (I need to make a style guideline for that too).